src/js/components/award/shared/activity/ContractGrantActivityChartCircles.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * ContractGrantActivityChartCircles.jsx
+ * Created by Jonathan Hill 04/23/2020
+ */
+
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { formatMoney } from 'helpers/moneyFormatter';
+
+const propTypes = {
+    transactions: PropTypes.array,
+    padding: PropTypes.object,
+    xScale: PropTypes.func,
+    yScale: PropTypes.func,
+    xAxisSpacing: PropTypes.number,
+    height: PropTypes.number
+};
+
+const ContractGrantActivityChartCircles = ({
+    transactions,
+    padding,
+    xScale,
+    yScale,
+    xAxisSpacing,
+    height
+}) => {
+    // circle data
+    const [circleData, setCircleData] = useState([]);
+    // sets circle data - hook - on mount and when transactions change
+    useEffect(() => {
+        if (xScale && yScale) {
+            const circles = transactions
+                .map((data, i) => ({
+                    key: `${data.federal_action_obligation}${i}`,
+                    description: `A circle representing the transaction date, ${data.action_date.format('MM-DD-YYYY')} and running total obligation of ${formatMoney(data.running_obligation_total)}`,
+                    className: 'transaction-date-circle',
+                    cx: xScale(data.action_date.valueOf()) + padding.left,
+                    cy: (height - yScale(data.running_obligation_total)),
+                    r: 1.5
```

Comment:
What is this? Maybe give a better name? Obviously its the same as the `circle` dom element, but might be helpful to give a different name here.

Reply:
r is radius.
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/circle